cirq/circuits/optimization_pass.py

```diff
@@ -41,8 +44,14 @@ def __init__(self, clear_span: int, clear_qubits: Iterable['cirq.Qid'],
                 with each affected moment.
             new_operations: The operations to replace the cleared out
                 operations with.
+            preserve_moments: Whether to keep Moments intact instead of
+                flattening them in this summary. Please be advised that
+                a PointOptimizer consuming this summary will flatten
+                operations no matter what, see gh-2406.
```

Comment:
Use a full link instead of the abbreviation.

Reply:
done